source/features/mark-merge-commits-in-list.tsx

```diff
@@ -39,7 +39,7 @@ export function getCommitHash(commit: HTMLElement): string {
 
 async function init(): Promise<void> {
 	const pageCommits = select.all('li.js-commits-list-item');
-	const mergeCommits = await filterMergeCommits(pageCommits.map(getCommitHash));
+	const mergeCommits = await filterMergeCommits(pageCommits.map(element => getCommitHash(element)));
```

Comment:
pageCommits -> commit
```suggestion
	const mergeCommits = await filterMergeCommits(pageCommits.map(commit => getCommitHash(commit)));
```